sfepy/terms/terms_diffusion.py

```diff
@@ -92,7 +92,7 @@ class SDDiffusionTerm(Term):
         - material:    :math:`K_{ij}`
         - parameter_q: :math:`q`
         - parameter_p: :math:`p`
-        - parameter_v: :math:`\ul{\Vcal}`
+        - parameter_mesh_velocity: :math:`\ul{\Vcal}`
```

Comment:
Why this renaming? `parameter_v` seems to be correct according to `arg_types` below.